src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,53 @@
+.watch {
+  position: relative;
```

Comment:
as i remember you cant use position for BEM block, but today Anna said that she not sure about 'position:relative'. So dont think it`s critical.

Reply:
that is the quastion)